### PR TITLE
Gnome fixes (gnome-documents and eog)

### PR DESCRIPTION
**Pull request by hedning**
###### Motivation for this change

Fixes gnome-documents so it can actually show documents.

Fixes eog thumbnailing.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/desktops/gnome-3/apps/gnome-documents/default.nix [hunk 1]
```diff
@@ -31,6 +31,11 @@ stdenv.mkDerivation rec {
     gnome-desktop libzapojit libgepub
   ];
 
+  patches = [
+    # fix RPATH to libgd
+    ./libgd.patch
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/apps/gnome-documents/default.nix]: Could you fetch https://gitlab.gnome.org/GNOME/gnome-documents/commit/d18a92e0a940073ac766f609937539e4fc6cdbb7.patch?

Comment (hedning) [reply pkgs/desktops/gnome-3/apps/gnome-documents/default.nix]: Can do :+1:

**Review by jtojnar — APPROVED** [review]

**Comment by jtojnar** [comment]
Relevant, GNOME Documents might be dropped https://mail.gnome.org/archives/desktop-devel-list/2019-January/msg00024.html

**Comment by hedning** [comment]
Right, lets consider removing it in 3.32 then (at least from the default).

**Pull request merged**